src/host/ut_host/Utf16ParserTests.cpp

```diff
@@ -1,211 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-#include "precomp.h"
-#include "WexTestClass.h"
-#include "../../inc/consoletaeftemplates.hpp"
-
-#include "../../types/inc/Utf16Parser.hpp"
-
-using namespace WEX::Common;
-using namespace WEX::Logging;
-using namespace WEX::TestExecution;
-
-static const std::vector<wchar_t> CyrillicChar = { 0x0431 }; // lowercase be
-static const std::vector<wchar_t> LatinChar = { 0x0061 }; // uppercase A
-static const std::vector<wchar_t> FullWidthChar = { 0xFF2D }; // fullwidth latin small letter m
-static const std::vector<wchar_t> GaelicChar = { 0x1E41 }; // latin small letter m with dot above
-static const std::vector<wchar_t> HiraganaChar = { 0x3059 }; // hiragana su
-static const std::vector<wchar_t> SunglassesEmoji = { 0xD83D, 0xDE0E }; // smiling face with sunglasses emoji

```

Comment:
Do we still want this test coverage in UnicodeTests.cpp? They'd pass, but idk if there's any value to specifically having a test for different chars like this.

Reply:
I'm not entirely sure what the purpose of these specific characters was, since UTF16 itself doesn't really care about Unicode assignments. So that's why in the new unit tests I'm just using a few random representative characters.